Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -440,6 +440,7 @@ FullDiagnostics::PrepareFieldDataForOutput ()
     warpx.FillBoundaryAux(warpx.getngUpdateAux());
```

Comment:
Note: this call to `FillBoundaryAux` before `UpdateAuxilaryData` does not seem intuitive. Would it be safe to remove it? Does it make a difference in specific scenarios, for example, in simulations with more than one refinement level?

Reply:
Yes, I will try to incorporate these changes within a separate PR, so that it is easier to assess their effects. In the meantime, I think the calls to `FillBoundaryAux` before `UpdateAuxilaryData` were introduced [here](https://github.com/ECP-WarpX/WarpX/pull/504/files#diff-140169a8e394017bac0dd16e554a7fd7a9f4aa5adfb0991fe37f821b395398d2).